extension/content/components/recipes/form/filters/partitionFO.js

```diff
@@ -0,0 +1,21 @@
+export const partitionFO = (filter_object) => {
+  const KNOWN_FILTER_TYPES = [
+    "channel",
+    "version",
+    "country",
+    "locale",
+    "bucketSample",
+    "stableSample",
+  ];
+
+  return filter_object.reduce(
+    ([knownFO, additionalFO], fo) => {
+      if (fo && KNOWN_FILTER_TYPES.includes(fo.type)) {
+        return [[...knownFO, fo], additionalFO];
+      }
+
+      return [knownFO, [...additionalFO, fo]];
+    },
+    [[], []],
+  );
+};
```

Comment:
I'm not sure why we need this function? Its use outside `FallbackFO` seem unnecessary?

Reply:
also if we do need it, this is not a component and is more in the shape of a utility function and so it should not live under `components` unless maybe it's in a single file that uses it like `FallbackFO.js` as a helper function that is not exported.

Reply:
if we do decide to keep this function I think it should return an object roughly in the shape of:
```js
{
  known: [],
  unknown: [],
}
```

or something to that effect rather than an array as that's less opaque

Reply:
oooh you're right it isnt needed, this lingered from my first attempt where i tried to save the json blob regardless of whether it was valid or not, so FO was an array of objects and strings and it got weird... didn't go back to check if it was still needed my bads